app/models/category.rb

```diff
@@ -0,0 +1,4 @@
+class Category < ApplicationRecord
+  belongs_to :category
```

Comment:
ac cho e hỏi. ở model này làm sao để thể hiện được relation đệ quy ạ.

Reply:
hỏi google có ngay mà chú
https://www.leighhalliday.com/tree-structures-in-your-rails-models
https://guides.rubyonrails.org/association_basics.html#self-joins

Reply:
> hỏi google có ngay mà chú
> https://www.leighhalliday.com/tree-structures-in-your-rails-models
> https://guides.rubyonrails.org/association_basics.html#self-joins

thanks c. e k biết key nào để search  (fank)

Reply:
hehe, nó là kỹ thuật join cảu DB - self join đấy chú nhé